i-100 to i-96 added for issue #18

pythagoraswitch opened the pull request:
Notable changes: 
- "source" is now corresponding url from 100 indicators site, i.e.  http://indicators.report/indicators/i-96/
- “available” section follows A-C assessment from “Preliminary
  assessment of current data availability by Friends of the Chair” from
  indicators.report site.
- i-7 url does not follow the http://indicators.report/indicators/i-**/ rule:
  indicators.report/indicators/7-total-fertility-rate/

pythagoraswitch committed:
i-100 to i-96 added for issue #18

Notable change: the source is now corresponding url from 100 indicators site, i.e.  http://indicators.report/indicators/i-96/

brunosan commented on indicators.json:
```diff
@@ -3618,33 +3618,33 @@
       },
-      "source": "http://unsdsn.org/wp-content/uploads/2014/02/140214-SDSN-indicator-report-DRAFT-for-consultation.pdf",
-      "leads": " OECD",
+      "source": "http://indicators.report/indicators/i-96/",
```
Good call on putting the right `source`. I'll do that on #19 too.

pythagoraswitch committed:
i-95 to i-51 added for issue #18

pythagoraswitch committed:
fix for the commit eb19122

pythagoraswitch committed:
i-50 to i-1 added/modified for issue #18

Notes: “available” section follows A-C assessment from “Preliminary
assessment of current data availability by Friends of the Chair” from
indicators.report.
i-7 url does not follow the rule:
indicators.report/indicators/7-total-fertility-rate/

brunosan commented:
Sweet! Thanks. :shipit:
